bees/ircbee/ircbee.go

```diff
@@ -156,6 +156,55 @@ func (mod *IrcBee) Run(eventChan chan bees.Event) {
 	mod.client.HandleFunc("disconnected", func(conn *irc.Conn, line *irc.Line) {
 		mod.connectedState <- false
 	})
+
+	statusChange := func (conn *irc.Conn, line *irc.Line) {
+		//Line.CMD eq Handler Name ex: JOIN
+		message := ""
+		switch line.Cmd {
+		case "JOIN":
+				message = "User Joined to channel " + line.Args[0]
+		case "PART":
+				message = "User Joined to channel " + line.Args[0]
+		case "QUIT":
+				message = "User Quit from channel " + line.Args[0]
+		}
+		mod.Logln(message)
+		channel := line.Args[0]
+		if channel == mod.client.Config().Me.Nick {
```

Comment:
Since we're only handling channel related events and you can't join/part a query, this can probably be removed.